### PR TITLE
Enable multipath for image xfer also on cinder-volume

### PR DESCRIPTION
**Pull request by dirkmueller**
Without the settings on Nova and Cinder-volume in sync, there are
interesting side effects in several drivers since they pass through
inconsistent information between Cinder and Nova, that causes undesired
artefacts.

### CODE REVIEW

**Comment by vuntz** [comment]
Can you add a comment to the nova cookbook about keeping this in sync?

**Comment by vuntz** [comment]
+1

File: chef/cookbooks/nova/attributes/default.rb [hunk 1]
```diff
@@ -36,6 +36,8 @@
 # Hypervisor Settings
 #
 default[:nova][:libvirt_type] = "kvm"
+
+# Keep in sync with cinder cookbook
 if node[:platform_family] == "suse"
   default[:nova][:libvirt_use_multipath] = true
```
Comment (tboerger) [inline chef/cookbooks/nova/attributes/default.rb]: No else branch here?

Comment (dirkmueller) [inline chef/cookbooks/nova/attributes/default.rb]: the else branch is on line 43 ?

**Comment by vuntz** [comment]
Anyone else for a +1?

**Comment by tboerger** [comment]
:+1:

**Pull request merged**